tests/cloud_functions/token_refresh/conftest.py

```
@@ -0,0 +1,26 @@
+        headers = {"Authorization": "Bearer test_token"}
+        url = "fake_url"
+        args = {"data_source": "strava"}
+
+        def get_json(self):
+            return {}
+
+    return MockRequest()
+
+
+@pytest.fixture
+def mock_decoded_token():
```

Ruff (0.8.2)

Missing return type annotation for public function `mock_decoded_token`

(ANN201)